backend/orchestrator/libs/models/serviceEvent.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity } from 'dynamodb-toolbox';
+
+import { PARTITION_KEY, SORT_KEY } from '../../resources/dynamodb';
+import OrchestratorTable from '../orchestratorTable';
+
+const ServiceEventEntity = new Entity({
+  name: 'ServiceEvent',
+  attributes: {
+    [PARTITION_KEY]: {
+      partitionKey: true,
+      hidden: true,
+      default: ({
+        serviceId,
+        applicationId,
+      }: {
+        serviceId: string;
+        applicationId: string;
+      }) => `${applicationId}#SERVICE#${serviceId}`,
+    },
+    [SORT_KEY]: {
+      sortKey: true,
+      hidden: true,
+      default: ({ timestamp }: { timestamp: string }) => `EVENT#${timestamp}`,
+    },
+    serviceId: { type: 'string', required: true },
+    applicationId: { type: 'string', required: true },
+    timestamp: { type: 'string', required: true },
```

Comment:
Instead of maintaining the timestamp ourselves, we can leverage [dynamodb-toolbox `created` attribute](https://github.com/jeremydaly/dynamodb-toolbox#specifying-entity-definitions)

Reply:
That is right, I replaced it with an `eventId` that is an ulid